e2e-tests/tests/config-router/06-restart-router.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      kubectl -n "${NAMESPACE}" rollout restart deployment config-router-router
```

Comment:
In PXC we need just create <cluster-name>-pxc cm and all pxc pods will be restarted automatically. Can we do the same here?

Reply:
Maybe, not sure, need to check it.

Reply:
> Maybe, not sure, need to check it.
> 
> But @hors what about when you update CM? You will still need to restart by hand right?

as I remember no

Reply:
In that case, I don't see a big value to have it TBH.

Reply:
We agreed, we will do this for mysql and router in a separate PR/task.